fiftyone/utils/hrm2.py

```
@@ -0,0 +1,2243 @@
+            self.export_meshes = False
+
+        if self.export_meshes and self._device is None:
+            logger.warning(
+                "export_meshes=True but no device provided. "
+                "This may cause issues during scene metadata preparation."
+            )
+
+    def __call__(
+        self,
+        outputs: List[Dict[str, Any]],
+        frame_size: Tuple[int, int],
```

Ruff (0.14.5)

Unused method argument: `frame_size`

(ARG002)